File: native/mixdog-graph/tests/fixtures/calls/ruby/sample.rb
# hidden()
quoted = "hidden()"

def inner; end
def leaf(x); x; end
def nest(x); x; end
def helper; end
def seed; end
def plain; end

plain()

def run(a)
  inner()
  nest(leaf(1))
  a.b().c()
end

class Widget
  def ping; end
  def act
    helper()
    self.ping()
  end
end

mark = "μ"; seed()
Widget.new()

File: native/mixdog-graph/tests/fixtures/calls/dart/sample.dart
// hidden()
var quoted = "hidden()";

void inner() {}
int leaf(int x) => x;
void nest(int x) {}
void helper() {}
void seed() {}
int plain() => 0;

class Chain {
  Chain b() => this;
  Chain c() => this;
}

var g = plain();

void run(Chain a) {
  inner();
  nest(leaf(1));
  a.b().c();
}

class Widget {
  void ping() {}
  void act() {
    helper();
    this.ping();
  }
}

void seedLine() {
  var mark = "μ"; seed();
  var w = new Widget();
}
